pythonforandroid/bootstrap.py

```diff
@@ -263,11 +263,8 @@ def strip_libraries(self, arch):
             info('Python was loaded from CrystaX, skipping strip')
             return
         env = arch.get_env()
-        strip = which('arm-linux-androideabi-strip', env['PATH'])
-        if strip is None:
-            warning('Can\'t find strip in PATH...')
-            return
-        strip = sh.Command(strip)
+        tokens = env['STRIP'].split(' ')
+        strip = reduce(lambda cmd, t: cmd.bake(t), tokens, sh.Command(tokens.pop(0)))
```

Comment:
Ok I played around with this, and it appears this can be written as:

```
tokens = env['STRIP'].split(' ')
strip = sh.Command(tokens[0]).bake(tokens[1:])
```
I would suggest that would be a little more readable

(This also works when `len(token) == 1` and consequently `tokens[:1] == []`, I tested that special case)

Sorry I really didn't want to go on about this forever from my initial duplicate option nitpick, I hope I'm not wearing you out too much :grimacing: but the reduce/lambda variant is a little hard to read

Reply:
Additional note: as written above I just noticed the space char splitting could break things.
```
tokens = shlex.split(env['STRIP'])
strip = sh.Command(tokens[0]).bake(tokens[1:])
```
This should work correctly (requires `import shlex` at the beginning of the file, which is a core/stdlib module as far as I'm aware)